rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -765,9 +765,14 @@ rm -f %{_builddir}/file.list.%{name}
 %dir %{rudderlogdir}/install
 
 %if 0%{?rhel} != 3 && "%{?_os}" != "aix"
-%config(noreplace) /etc/ld.so.conf.d/rudder.conf
+%config /etc/ld.so.conf.d/rudder.conf
 %endif
 
+%config /etc/cron.d/rudder-agent
+%config /etc/profile.d/rudder-agent.sh
```

Comment:
I don't see why we are changing these in a released versoin - does this fix a bug? I'm fine with doing it in master just not on 2.11 unless we have a reason.